src/Makefile.am

```diff
@@ -377,6 +381,9 @@ bitcoind_LDADD += $(BOOST_LIBS) $(BDB_LIBS) $(SSL_LIBS) $(CRYPTO_LIBS) $(MINIUPN
 
 # bitcoin-cli binary #
 bitcoin_cli_SOURCES = bitcoin-cli.cpp
+if BUILD_EVUNIX
```

Comment:
I think this is OK. I think we'll end up with a few wrappers for libevent, so moving those to their own internal helper lib will  make sense.